src/ParallelAlgorithms/DiscontinuousGalerkin/InitializeMortars.hpp

```diff
@@ -177,6 +169,39 @@ struct InitializeMortars {
         "Dependencies not fulfilled. Did you forget to terminate the phase "
         "after removing options?");
   }
+
+ private:
+  template <typename DbTagsList>
+  static auto initialize_next_temporal_id_on_mortars(
+      db::DataBox<DbTagsList>&& box, std::true_type /* meta */) noexcept {
```

Comment:
instead of meta can you replaces with is_bla. (where bla is whatever is true...)

Reply:
named `enabled` and `disabled`, since the function name already describes what is being enabled or disabled